Makefile

```diff
@@ -32,25 +32,24 @@ GIT_TAG    = $(shell git describe --tags --abbrev=0 --exact-match 2>/dev/null)
 GIT_DIRTY  = $(shell test -n "`git status --porcelain`" && echo "dirty" || echo "clean")
 
 ifdef VERSION
-	BINARY_VERSION = $(VERSION)
+BINARY_VERSION = $(VERSION)
+else
+BINARY_VERSION = $(GIT_TAG)
 endif
-BINARY_VERSION ?= ${GIT_TAG}
 
 # Only set Version if building a tag or VERSION is set
 ifneq ($(BINARY_VERSION),)
-	LDFLAGS += -X helm.sh/helm/v3/internal/version.version=${BINARY_VERSION}
+VERSION_LDFLAGS += -X helm.sh/helm/v3/internal/version.version=$(BINARY_VERSION)
```

Comment:
Out of curiosity, why did you remove the indentation of the body of the conditionals?

Reply:
I believe it's more idiomatic to not indent conditionals - see https://www.gnu.org/software/make/manual/html_node/Conditional-Syntax.html for examples that show no indentation.

Reply:
The conditional examples in the manual have an indent. See https://www.gnu.org/software/make/manual/html_node/Conditional-Example.html#Conditional-Example. I was looking at those two pages before I made my comment because I wanted to find out what was idiomatic. I couldn't tell because those two pages in the manual aren't consistent.

Do you have another piece of information to show that it's idiomatic to not indent? If not, to be consistent with what we have had I would ask to keep the indent. It's a little easier to read.

Reply:
@mattfarina Fair point. I've reverted the indentation.